components/match2/wikis/dota2/match_page_template.lua

```diff
@@ -63,19 +63,51 @@ return {
 					<div class="match-bm-lol-game-summary-team">{{&opponents.2.iconDisplay}}</div>
 				</div>
 			</div>{{/isBestOfOne}}
-			<h3>Picks and Bans</h3>
-			<div class="match-bm-lol-game-veto collapsed general-collapsible">
+			<h3>Draft</h3>
+			<div class="match-bm-game-veto-wrapper">
 				<div class="match-bm-lol-game-veto-overview">
-					<div class="match-bm-lol-game-veto-overview-team"><div class="match-bm-lol-game-veto-overview-team-header">{{&opponents.1.iconDisplay}}</div>
-						<div class="match-bm-lol-game-veto-overview-team-veto">
-							<ul class="match-bm-lol-game-veto-overview-pick" aria-labelledby="picks">{{#teams.1.picks}}<li class="match-bm-lol-game-veto-overview-item">{{vetoNumber}}{{&heroIcon}}<div class="match-bm-lol-game-veto-pick-bar-{{teams.1.side}}"></div></li>{{/teams.1.picks}}</ul>
-							<ul class="match-bm-lol-game-veto-overview-ban" aria-labelledby="bans">{{#teams.1.bans}}<li class="match-bm-lol-game-veto-overview-item">{{vetoNumber}}{{&heroIcon}}</li>{{/teams.1.bans}}</ul>
+					<div class="match-bm-lol-game-veto-overview-team">
+						<div class="match-bm-lol-game-veto-overview-team-header">{{&opponents.1.iconDisplay}}</div>
```

Comment:
Didn't change those as I reused the ones from league (same as [the previous PR](https://github.com/Liquipedia/Lua-Modules/pull/4508) from liquidely). I guess at some point we want to make those class more generic since we use them outside of lol

Reply:
Talked to Ely about it last week, she said she only added non-`lol` (generic) version where a change was needed compared to lol.
Once it's done we should transport all the ones we want to keep to generic, and remove all lol. As the post-golive-plan is to change the league layout to match that of dota2

Reply:
Yep, I followed the same strategy for this one